ra2ce/graph/networks.py

```diff
@@ -552,7 +639,10 @@ def create(self) -> dict:
             # Create the network from the network source
             if self.config["network"]["source"] == "shapefile":
                 logging.info("Start creating a network from the submitted shapefile.")
-                base_graph, network_gdf = self.network_shp()
+                if self.cleanup is False:
```

Comment:
```suggestion
                if not self.cleanup:
```
Although I would do it the other way around, `if self.cleanup: .... else: ... `